docs/website/docs/intro.md

```diff
@@ -10,112 +10,146 @@ import snippets from '!!raw-loader!./intro-snippets.py';
+
+dlt is designed to be easy to use, flexible, and scalable:
+
+- dlt infers [schemas](./general-usage/schema) and [data types](./general-usage/schema/#data-types), [normalizes the data](./general-usage/schema/#data-normalizer), and handles nested data structures.
+- dlt supports variety of [popular destinations](./dlt-ecosystem/destinations/) and has an interface to add [custom destinations](./dlt-ecosystem/destinations/destination) to create reverse ETL pipelines.
+- Use dlt locally or [in the cloud](./walkthroughs/deploy-a-pipeline) to build data pipelines, data lakes, and data warehouses.
+
+To get started with dlt, install the library using pip:
 
-`dlt` is an open-source library that you can add to your Python scripts to load data
-from various and often messy data sources into well-structured, live datasets. To get started, install it with:
 ```sh
 pip install dlt
 ```
 :::tip
-We recommend using a clean virtual environment for your experiments! Here are [detailed instructions](/reference/installation).
+We recommend using a clean virtual environment for your experiments! Here are [detailed instructions](/reference/installation) on how to set up one.
 :::
 
-Unlike other solutions, with dlt, there's no need to use any backends or containers. Simply import `dlt` in a Python file or a Jupyter Notebook cell, and create a pipeline to load data into any of the [supported destinations](dlt-ecosystem/destinations/). You can load data from any source that produces Python data structures, including APIs, files, databases, and more. `dlt` also supports building a [custom destination](dlt-ecosystem/destinations/destination.md), which you can use as reverse ETL.
-
-The library will create or update tables, infer data types, and handle nested data automatically. Here are a few example pipelines:
+## Load data with dlt from …
 
 <Tabs
   groupId="source-type"
-  defaultValue="api"
+  defaultValue="rest-api"
   values={[
-    {"label": "Data from an API", "value": "api"},
-    {"label": "Data from a dlt Source", "value": "source"},
-    {"label": "Data from CSV/XLS/Pandas", "value": "csv"},
-    {"label": "Data from a Database", "value":"database"}
+    {"label": "REST APIs", "value": "rest-api"},
+    {"label": "SQL databases", "value": "sql-database"},
+    {"label": "Cloud storages or files", "value": "filesystem"},
+    {"label": "Python data structures", "value": "python-data"},
 ]}>
-  <TabItem value="api">
+  <TabItem value="rest-api">
 
-:::tip
-Looking to use a REST API as a source? Explore our new [REST API generic source](dlt-ecosystem/verified-sources/rest_api) for a declarative way to load data.
-:::
+Use dlt's [REST API source](tutorial/rest-api) to extract data from any REST API. Define API endpoints you’d like to fetch data from, pagination method and authentication and dlt will handle the rest:
 
-<!--@@@DLT_SNIPPET api-->
+```py
+# from dlt.sources import rest_api
```

Comment:
why are these commented out?

Reply:
This was before we moved the rest_api source to the core so I commented it for docs tests to pass